.rubocop.yml

```
@@ -190,18 +337,57 @@ Rails/HelperInstanceVariable:
 Rails/HttpStatus:
   Enabled: false
 
-Rails/SaveBang:
+Rails/MailerName:
   Enabled: true
 
-# read_attribute(:attr) and self[:attr] are no longer equivalent
-Rails/ReadWriteAttribute:
-  Enabled: false
+Rails/MatchRoute:
+  Enabled: true
+
+Rails/NegateInclude:
+  Enabled: true
 
 # post_scraper.rb is run manually in the command line so stdout is fine
 Rails/Output:
   Exclude:
     - 'app/services/post_scraper.rb'
 
+Rails/Pluck:
+  Enabled: true
+
+Rails/PluckInWhere:
+  Enabled: false
+
+# read_attribute(:attr) and self[:attr] are no longer equivalent
+Rails/ReadWriteAttribute:
+  Enabled: false
+
+Rails/RenderInline:
+  Enabled: true
+
+Rails/RenderPlainText:
+  Enabled: true
+
+Rails/SaveBang:
+  Enabled: true
+
+Rails/ShortI18n:
+  Enabled: true
+
+Rails/SquishedSQLHeredocs:
+  Enabled: false
+
+Rails/UniqueValidationWithoutIndex :
```

Weird space on this one (was this config auto-generated?)
```suggestion
Rails/UniqueValidationWithoutIndex:
```

---

I was copy-pasting from what rubocop gave me a lot but I think what one is actually my fault xD